deployment/distribution/src/main/kotlin/org/codice/ctk/Command.kt

```diff
@@ -90,8 +90,15 @@ private fun launchTests(arguments: ArgumentCollection) {
         registerTestExecutionListeners(summaryGeneratingListener)
     }.execute(request)
 
-    PrintWriter(System.out).use {
-        summaryGeneratingListener.summary.printFailuresTo(it)
-        summaryGeneratingListener.summary.printTo(it)
+    PrintWriter(System.out).use { printer ->
+        summaryGeneratingListener.summary.printFailuresTo(printer)
+        summaryGeneratingListener.summary.printTo(printer)
+
+        if (summaryGeneratingListener.summary.totalFailureCount > 0) {
+            System.out.println("TESTS FAILED")
```

Comment:
Tried using `printer.println` here instead of doing `System.out.println` but wasn't working